tsfc/coffee.py

```diff
@@ -296,7 +299,7 @@ def _expression_scalar(expr, parameters):
     if isnan(expr.value):
         return coffee.Symbol("NAN")
     else:
-        return coffee.Symbol(("%%.%dg" % (PRECISION - 1)) % expr.value)
+        return coffee.Symbol(("%%.%dg" % (parameters.precision - 1)) % expr.value)
```

Comment:
I could compare this with uflacs and quadrature. Do you a form which goes through this codepath?